docs/source/conf.py

```diff
@@ -694,6 +694,9 @@
     "probs_to_logits",
     "tril_matrix_to_vec",
     "vec_to_tril_matrix",
+    # torch.export.dynamic_shapes
+    "Dim",
```

Comment:
Please don't add new entries here. Just document the functions properly so that they show up on the docs website.
In this case you most likely are missing an entry in the rst file for these two

Reply:
But they are in the rst files :((